### PR TITLE
Updated CSA examples to new documentation guidelines. [ci skip]

### PR DESCRIPTION
**Pull request by jougs**
See trac.967 and trac.968 for details.

@heplesser: could you please review this?

### CODE REVIEW

File: pynest/examples/csa_example.py [hunk 1]
```diff
@@ -19,57 +19,96 @@
...
+and two values (10000.0 and 1.0) used as weight and delay,
+respectively.
+'''
 
-def csa_example():
+cs = csa.cset(csa.random(0.1), 10000.0, 1.0)
 
-    cs = csa.cset(csa.random(0.1), 10000.0, 1.0)
+'''
+The neurons of the pre- and postsynaptic populations are
```
Comment (heplesser) [inline pynest/examples/csa_example.py]: You are jumping back and forth between active "we create" and passive "neurons ... are created" throughout the text.

**Comment by heplesser** [comment]
@jougs Looks fine to me, except for your variation between active and passive. I have not run the examples. Conditional :+1:.

**Comment by jougs** [comment]
@heplesser: I've added some new commits and hope that they address your concerns.

**Comment by heplesser** [comment]
:+1:

**Pull request merged**